app.js

```diff
@@ -6,7 +6,7 @@ app.use(express.json()); // app.use handles specific requests - here we are pars
 
 app.get("/api/topics", fetchTopics); // defines a GET route, When a GET request made ,call the fetchTopics function made in controller file
 
-app.get("/*", (req, res) => {
+app.all("/*", (req, res) => {
```

Comment:
This is the only error we need for this endpoint but take a look at the comment on your controller